scripts/st2bootstrap-el7.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -eux
```

Comment:
This is expected to exit on error, from my point of view.

I tried Ubuntu script and it doesn't catch any errors at all.
Even if any command fails - script marked as succeeded. Not sure if that's expected.